src/sykmeldt/components/oppfolgingsplan/godkjenn/TilbakeLenke.tsx

```diff
@@ -0,0 +1,12 @@
+import { useLandingUrl } from "@/common/hooks/routeHooks";
+import Link from "next/link";
+
+export const TilbakeLenke = () => {
+  const landingPage = useLandingUrl();
+
+  return (
+    <div>
+      <Link href={landingPage}>Tilbake til oppfølgingsplaner</Link>
```

Comment:
lurer på om for å få med all riktig aria-ting osv, så bør vi sette passHref={true} på Link komponenten, og bruke en designsystem-link inni.

Kan feks kalle next-linken for NextLink i imports